news/5128.hooks.rst

```diff
@@ -0,0 +1 @@
+Update ``sqlalchemy`` hook to support v1.3.19 and later,  by adding ``pysqlite2``, ``MySQLdb`` and ``psycopg2`` as hidden imports.
```

Comment:
```suggestion
Update ``sqlalchemy`` hook to support v1.3.19 and later,  by adding ``sqlalchemy.ext.baked`` as a hidden import
```

Reply:
```suggestion
Update ``sqlalchemy`` hook to support v1.3.19 and later.
```

Reply:
Take whichever you like @RS357; it doesn't matter much.